src/SFA.DAS.Payments.ProviderPayments.Application/Services/CollectionPeriodStorageService.cs

```diff
@@ -40,15 +40,14 @@ await context.CollectionPeriod.AddAsync(new CollectionPeriodModel
             await context.SaveChanges();
         }
 
-        private async Task<DateTime?> GetReferenceDataValidationDate(short academicYear, byte period)
+        private DateTime? GetReferenceDataValidationDate(short academicYear, byte period)
         {
-            var job = await context.Job.Where(x => x.JobType == JobType.PeriodEndSubmissionWindowValidationJob
-                                                && x.AcademicYear == academicYear
-                                                && x.CollectionPeriod == period
-                                                && x.EndTime != null
-                                                && x.EndTime.Value != null)
+            var job = context.Job.Local.Where(x => x.JobType == JobType.PeriodEndSubmissionWindowValidationJob
```

Comment:
My understanding is that using .Local means we don't query the SQL database, only the in memory copy, is that correct?

Reply:
Also to note, the unit tests were failing due to the restructuring of the DBSet model, as no entities were being found at `context.Job.Where` level